### PR TITLE
Www 84 improve responsive layout on components

### PR DESCRIPTION
**Pull request by scastillo-jp**
Screen Shot 2020-07-20 at 5 55 15 PM

### CODE REVIEW

**Review by JuanJosephG — APPROVED** [review]

File: src/components/atoms/RichTextWrapper/RichTextWrapper.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React from "react"
 import theme from "../../theme"
-import { Typography } from "@material-ui/core"
+import { Typography, Container } from "@material-ui/core"
```
Comment (JuanJosephG) [inline src/components/atoms/RichTextWrapper/RichTextWrapper.js]: @scastillo-jp Container is defined but never used. You should remove it.

File: src/components/molecules/CheckerboardRow/CheckerboardRow.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  .MuiGrid-root .MuiGrid-item  > img{
+    width: 392px;
+    height: 24.18rem;
+    object-fit: cover;
+    margin: 0;
+  }
+  p {
+    padding: 10px 15px 10px 10px;
+  }
+}
```
Comment (Wason1797) [inline src/components/molecules/CheckerboardRow/CheckerboardRow.scss]: please add a new line at the end of the file

File: src/components/molecules/MapCard/defaultStyles.js [hunk 3]
```diff
@@ -9,8 +9,9 @@ const defaultStyles = {
     background: "white",
     maxWidth: 345,
     maxHeight: "100%",
-    width: "",
-    height: "100%",
+    height: 526,
```
Comment (Wason1797) [inline src/components/molecules/MapCard/defaultStyles.js:13]: it was necessary to remove width?

**Review by Wason1797 — COMMENTED** [review]
Some minor comments

File: src/components/molecules/MapCard/defaultStyles.js (hunk 3 above)
Comment (scastillo-jp) [reply src/components/molecules/MapCard/defaultStyles.js:13]: no is necessary remove, I restored this  width: "",

**Review by JuanJosephG — APPROVED** [review]
Good job! 👍

**Review by Wason1797 — APPROVED** [review]
Nice Job!

**Pull request merged**